### PR TITLE
fix: add npm dep

### PR DESCRIPTION
**Pull request by zhaofeng-shu33**

### CODE REVIEW

**Comment by alvinhui** [comment]
0. make sure the npm version is match node@10.16.0;
0. this npm should be fallback, please check getEnv function and make sure fallback logic.

**Comment by zhaofeng-shu33** [comment]
0. is surely.

File: tools/iceworks-app/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "electron-updater": "^4.1.2",
     "execa": "^1.0.0",
     "ice-npm-utils": "^1.1.0",
+    "npm": "^6.9.0",
```
Comment (alvinhui) [inline tools/iceworks-app/package.json]: A better way is to have a fixed version number.

Comment (zhaofeng-shu33) [reply tools/iceworks-app/package.json]: OK

**Comment by alvinhui** [comment]
please also update `getEnv` in iceworks-server.

**Comment by zhaofeng-shu33** [comment]
`electron-build` will ignore `.bin` directory. Therefore, I add a symbol link directly to the js file.

**Pull request merged**